src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -158,6 +158,7 @@ public void addTermRating(HttpServletRequest request, DatastoreService datastore
     termRatingEntity.setProperty("comments-professor", professorFeedback);
     termRatingEntity.setProperty("score-professor", professorScore);
     termRatingEntity.setProperty("perception-professor", professorRating);
+    termRatingEntity.setProperty("translation", Boolean.toString(translate));
```

Comment:
What if someone types something in a different language but then doesn't check the translate button? Is there a way to detect if the comment is in a different language and automatically translate then?

Reply:
So, originally I was doing this to save on operation time and maybe if you automatically translated something from English to English it could change in some capacity if there was like slang or something so wanted to let them choose. Let's check with Luke and Travis on this to see what they thinkk :)

Reply:
Checked! I think its better if we keep this as it is for the moment (talked with Luke and Travis and right now is not a priority)